src/styles/stars.scss

```
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
```

```suggestion
.stars {
  display: flex;
  &__star {
```
Make use of SASS nesting to make your code cleaner